include/sdf/Error.hh

```diff
@@ -179,6 +179,12 @@ namespace sdf
 
     /// \brief Generic error during parsing.
     PARSING_ERROR,
+
+    /// \brief File not found error.
+    FILE_NOT_FOUND,
```

Comment:
Should this be `FILE_NOT_FOUND_ERROR` to match the convention of the others?

Reply:
In the interest of not changing enums as much as possible in stable versions, I'd say we should use the `FILE_READ` error code.

Reply:
this enum isn't used in this PR; I've opened https://github.com/gazebosim/sdformat/pull/1442 targeting https://github.com/gazebosim/sdformat/pull/1295 to implement this suggestion

Reply:
removed in https://github.com/gazebosim/sdformat/pull/1296/commits/2bc407d901162cac58581c9682e1f8c37c6f4c45